packages/paste-core/components/progress-steps/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import type {ProgressStepIncompleteProps} from './types';
+
+export const validateProps = (props: Pick<ProgressStepIncompleteProps, 'as' | 'href' | 'onClick'>): void => {
```

Comment:
suggestion: error for disabling an anchor step?

Reply:
Exists. Lives in the ProgressStepIncomplete file directly since it isn't shared behavior